<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mosaicDialog</class>
 <widget class="QDialog" name="mosaicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mosaic Job Creator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="equipmentGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Equipment</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>H</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="pixelHSizeSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>um</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>W</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="pixelWSizeSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>H</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="cameraHSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Focal Length:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Pixel Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Camera:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>W</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="cameraWSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>pixels</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="focalLenSpin">
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Rotation:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QDoubleSpinBox" name="rotationSpin">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>E of N</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fovGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>FOV</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="cameraFOVLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Camera FOV:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Mosaic:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="overlapSpin">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <double>20.000000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="cameraFOVUnits">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>arcmin</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="targetFOVLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Target FOV:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="targetFOVUnits">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>arcmin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="cameraHFOVLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>H</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="cameraHFOVSpin">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>3600.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="cameraWFOVLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>W</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="cameraWFOVSpin">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>3600.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>W</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mosaicWSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="overlapUnits">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>H</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mosaicHSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="overlapLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Overlap:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="targetFOVHLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>H</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="targetHFOVSpin">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>3600.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="targetFOVWLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>W</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="targetWFOVSpin">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>3600.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="jobCountLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Job Count:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="jobCountSpin">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="textLabel1_18">
              <property name="toolTip">
               <string>Directory to save sequence images</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>Directory:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="jobsDir"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectJobsDirB">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="updateB">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetB">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="createJobsB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Create Jobs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="mosaicView">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="Dense6Pattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>focalLenSpin</tabstop>
  <tabstop>cameraWSpin</tabstop>
  <tabstop>cameraHSpin</tabstop>
  <tabstop>pixelWSizeSpin</tabstop>
  <tabstop>pixelHSizeSpin</tabstop>
  <tabstop>cameraWFOVSpin</tabstop>
  <tabstop>cameraHFOVSpin</tabstop>
  <tabstop>targetWFOVSpin</tabstop>
  <tabstop>targetHFOVSpin</tabstop>
  <tabstop>overlapSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mosaicDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mosaicDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
